src/storage-client/src/controller/rehydration.rs

```diff
@@ -281,11 +281,15 @@ where
                 }
             };
 
+            // The first epoch we actually send to the cluster will be `1`, just like compute.
+            let new_epoch = ClusterStartupEpoch::new(
```

Comment:
We do also verify this epoch business for storage clusters because that code is shared/generic now, right?

https://github.com/MaterializeInc/materialize/blob/cd31ae1cb71af9adccb0bac67f8aac851eacc1d5/src/cluster/src/server.rs#L291